### PR TITLE
Files added:Online_Shopping_Site

### PR DESCRIPTION
**Pull request by shubhali-github**
Added new project Online shopping site

### CODE REVIEW

File: shubhali/Online_Shopping_Site/css/slide.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+/* Responsive layout - when the screen is less than 800px wide, make the two columns stack on top of each other instead of next to each other (also change the direction - make the "cart" column go on top) */
+@media (max-width: 800px) {
+    .row {
+    flex-direction: column-reverse;
+    }
+    .col-25 {
+      margin-bottom: 20px;
+    }
+  }
+
```
Comment (anitapise) [inline shubhali/Online_Shopping_Site/css/slide.css:71]: remove extra line from end of the file

File: shubhali/Online_Shopping_Site/css/style2.css [hunk 2]
```diff
@@ -98,7 +97,6 @@ input.email, input.text.w3lpass {
     visibility: visible !important;
 }
 :-webkit-input-placeholder {
-
```
Comment (Pragati-3) [inline shubhali/Online_Shopping_Site/css/style2.css]: Remove the extra spaces

File: shubhali/Online_Shopping_Site/login.html [hunk 3]
```diff
@@ -71,17 +71,14 @@ <h2>Login Form</h2>
             <div class="container">
                 <label for="uname"><b>Username</b></label>
                 <input type="text" placeholder="Enter Username" name="uname" required>
-
                 <label for="psw"><b>Password</b></label>
                 <input type="password" placeholder="Enter Password" name="psw" required>
-
```
Comment (Pragati-3) [inline shubhali/Online_Shopping_Site/login.html]: Remove the extra line

File: shubhali/Online_Shopping_Site/register.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+                                        <div class="modal-dialog">
+                                            <div class="modal-content" style="color: black;">
+                                                <!-- Modal Header -->
+                                                <div class="modal-header">
+                                                    <h4 class="modal-title">Terms and Conditions</h4>
+                                                    <button type="button" class="close" data-dismiss="modal">&times;</button>
+                                                </div>
+                                                <!-- Modal body -->
+                                                <div class="modal-body" style="color: black;">
+                                                    All products are subject to availability at the shop where yoy collect your order.Some shops do not sell all products.
```
Comment (anitapise) [inline shubhali/Online_Shopping_Site/register.html:50]: spelling mistake yoy make it you

File: shubhali/Online_Shopping_Site/register.html [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+                                                    Images of product and packaging on the online service are exampleonly and may not be identical to the product or packaging you receive from a shops.
```
Comment (anitapise) [inline shubhali/Online_Shopping_Site/register.html:51]: exampleonly space required example only

File: shubhali/Online_Shopping_Site/register.html [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+                                                    Difference may be due to your device's of colorsfactors such as the ingrediants used.the supplier,the region of thr country. and the season of the year.
```
Comment (anitapise) [inline shubhali/Online_Shopping_Site/register.html:52]: thr make it the

ingrediants used.the supplier (please check spelling and after full stop new sentence please make it correct)

**Pull request merged**